workflow/djworker.py

```diff
@@ -0,0 +1,150 @@
+import datajoint as dj
+import inspect
+import time
+import os
+import platform
+from datetime import datetime
+
+
+_populate_settings = {
+    'display_progress': True,
+    'reserve_jobs': True,
+    'suppress_errors': True}
+
+
+class WorkerLog(dj.Table):
+    definition = """
+    # Registration of processing jobs running .populate() jobs (e.g. in process.py script)
+    process_timestamp : datetime      # timestamp of the processing job
+    process           : varchar(64)
+    ---
+    worker_name=''    : varchar(255)  # name of the worker
+    host              : varchar(255)  # system hostname
+    user=''           : varchar(255)  # database user
+    pid=0             : int unsigned  # system process id  
+    """
+
+    _table_name = '~WorkerLog'
```

Comment:
we use lower snake case for all table names
```suggestion
    _table_name = '~worker_log'
```